bpf/process/bpf_execve_event.c

```diff
@@ -269,6 +269,26 @@ execve_rate(void *ctx)
 	return 0;
 }
 
+// update bitset mark
+FUNC_INLINE
+void update_mb_bitset(struct execve_map_value *curr)
```

Comment:
could you make it more obvious that it's for the `followChildren` feature?

Reply:
https://github.com/cilium/tetragon/pull/2720#discussion_r1698160468 should make it clear, I believe.